en/SetupView/Motors.md

```diff
@@ -1,23 +1,30 @@
 # Motors Setup
 
-This section is used to setup and test individual motors/servos, and applies to both ArduPilot and PX4:
+Motor Setup is used to test individual motors/servos (for example, to verify that motors spin in the correct direction).
```

Comment:
Note, I assume that you can't use this for testing that motor low throttle is set properly? When you update the PX4 docs I guess we'll need to make that distinction.

Reply:
Yes, exactly, that's the drawback with keeping the motors off at minimum slider value.